modules/gapi/src/backends/fluid/gfluidcore.cpp

```diff
@@ -863,20 +1583,37 @@ static void run_bitwise2(Buffer &dst, const View &src1, const View &src2, Bitwis
     int chan   = dst.meta().chan;
     int length = width * chan;
 
+    int x = 0;
+
     switch (bitwise_op)
     {
     case BW_AND:
-        for (int l=0; l < length; l++)
-            out[l] = in1[l] & in2[l];
+    {
+#if CV_SIMD
+        and_simd(in1, in2, out, length, x);
+#endif
+        for (; x < length; ++x)
```

Comment:
Looks like vectorized version handle data tails on the own so scalar version could be under `#else ... #endif`

Reply:
> Looks like vectorized version handle data tails on the own so scalar version 

But there is the case when `nlanes < length`. Am I right that then the scalar version should be used? If yes, there shouldn't be `#else`

Reply:
Agree